metrics/web/docs/features/navigation/diagrams/navigation_class_diagram.puml

```diff
@@ -0,0 +1,105 @@
+@startuml navigation_class_diagram
+
+class MetricsApp {}
+
+package common.presentation {
+
+    package injector.widget {
+        class InjectionContainer {}
+    }
+
+    package navigation {
+
+        package route_configration {
+            class RouteName {
+            }
+
+            class RouteConfiguration {
+                + name : RouteName
+                + path : String
+                + authorizationRequired : bool
+                + nestedRouteInformationFactory : RouteConfigurationFactory
```

Comment:
There is no description for the nested pages in this document. It should be reviewed and designed, or just remove this for now so we can introduce navigation that doesn't support nested yet.